schemas/components/schemas/Tampering.yaml

```diff
@@ -0,0 +1,29 @@
+type: object
+additionalProperties: false
+required:
+  - result
+  - anomalyScore
+  - antiDetectBrowser
+properties:
+  result:
+    type: boolean
+    description: >-
+      true if tampering is identified based on an anomalyScore or antiDetectBrowser signal 
```

Comment:
It seems this branch is based on an older version of `main` that does not necessarily include recent PRs, for example, the [copy improvements merged yesterday here](https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/150/files).

Let's include them and double-check that we are not accidentally overwriting any recent changes to the schema.